inflector.go

```diff
@@ -196,7 +197,7 @@ func init() {
 			{`(?i)(hive)s$`, `$1`},
 			{`(?i)(drive)s$`, `$1`},
 			{`(?i)(^analy)ses$`, `${1}sis`},
-			{`(?i)(analy|diagno|^ba|(p)arenthe|(p)rogno|(s)ynop|(t)he)ses$`, `${1}${2}sis`},
+			{`(?i)(analy|diagno|^ba|parenthe|progno|synop|the)ses$`, `${1}sis`},
```

Comment:
This looks like an unsuccessful attempt to handle capital words - do we handle these well?

Reply:
The `(?i)` modifier makes the whole regex case-insensitive. This test succeeds:

```
{"Parentheses", "Parenthesis", true},
```

Reply:
I see so the `(p)` and `(s)` groups were pointless

Reply:
Yeah I couldn't guess the motivation. Asked an LLM too but it said it made no sense.